include/seqan/graph_msa/graph_align_tcoffee_library.h

```diff
@@ -77,23 +77,47 @@ namespace SEQAN_NAMESPACE_MAIN
  * @brief Segment matches from pairwise longest common subsequence comparison.
  *
  * @signature typedef Tag<LcsLibrary_> const LcsLibrary;
+ *
+ *
+ *
+ * @defgroup MultipleSequenceAlignmentTags Multiple Sequence Alignment Tags
+ * @brief Tags specifying types of multiple alignment.
+ *
+ * @tag MultipleSequenceAlignmentTags#OrdinaryAlignment
+ * @headerfile <seqan/graph_msa.h>
+ * @brief Ordinary alignment with quite small number of sequences.
+ *
+ * @signature typedef Tag<OrdinaryAlignment_> OrdinaryAlignment;
+ *
```

Comment:
Maybe DefaultAlignment to refer to the standard t-coffee? Ordinary sounds kind of reprobative.